Error: "_debug_" redefined [-Werror]

Hi,

I have the following error when I try to compile with options:
* `--enable-debug` and `--enable-miniconfig`:
* or `--enable-debug2` and `--enable-miniconfig`

```sh
cmdline.mini.c:32:0: error: "_debug_" redefined [-Werror]
 #define _debug_ 0
 ^
In file included from chilli.h:25:0,
                 from cmdline.mini.c:27:
debug.h:25:0: note: this is the location of the previous definition
 #define _debug_ 2
 ^
```

Works fine when  `--disable-debug`, `--disable-debug2` and `--enable-miniconfig` are set.

--- a/src/cmdline.mini.c
+++ b/src/cmdline.mini.c
@@ -29,7 +29,9 @@
 
 #include <getopt.h>
 
-#define _debug_ 0
+#ifndef _debug_
+  #define _debug_ 0
+#endif
 
 #define OPT_string  0
 #define OPT_integer 1

